build.gradle

```diff
@@ -931,6 +936,20 @@ task runQuickstart {
       if (conffile.exists())
         conffile.delete()
     }
+    //Setting _JAVA_OPTIONS to avoid using gradle settings which is 256m which was causing rdd to
+    // overflow on disk
+    Map<String,Object> env = new HashMap<String,Object>()
+    env.put("_JAVA_OPTIONS","-Xms256m -Xmx4g")
+    String quickStartResult = runScript("${snappyProductDir}/bin/spark-shell", workingDir,
+        [' --driver-memory ', '4g', '-i',
+         "${snappyProductDir}/quickstart/scripts/Quickstart.scala", '--master', 'local[1]'], env)
+    if (quickStartResult.toLowerCase().
+        contains('cannot meet the required performance')) {
+      throw new GradleException('Failed to assert required performance in Quickstart')
```

Comment:
What's this about? Looks like something which will be highly hardware dependent and not really useful. This can be some internal flag (say -PassertPerformance) for our local dev testing only and not for users in general.